frontend/src/pages/team/Devices/dialogs/DeviceCredentialsDialog.vue

```diff
@@ -1,24 +1,30 @@
 <template>
-    <ff-dialog ref="dialog" header="Device Credentials">
+    <ff-dialog ref="dialog" header="Device Configuration">
         <template v-slot:default>
             <form class="text-gray-800">
                 <template v-if="!hasCredentials">
                     <p>
-                        Are you sure you want to regenerate credentials for this device?
+                        Are you sure you want to regenerate configuration for this device?
                     </p>
                     <p class="mt-3 mb-6">
-                        The existing credentials will be reset and the device will not
-                        be able to reconnect until it has been given its new credentials.
+                        The existing configuration will be reset and the device will not
+                        be able to reconnect until it has been given its new configuration.
                     </p>
                 </template>
                 <template v-if="hasCredentials">
                     <p>
                         To connect your device to the platform, use the following
-                        credentials.
+                        configuration. This <code>device.yml</code> file will need to be moved to your device.
```

Comment:
```suggestion
                        configuration. This will need to be placed on your device.
```

Specifying device.yml and stating "file" might be misleading. The user could use the Copy To Clipboard method and paste it into the Web-UI of the device.  This is all listed in the docs.  So I suggest we stay a little vague here and rely on the docs?

Reply:
and this one Joe - thoughts?

The docs are linked to in the very next sentence of this section, it ends up reading:
```html
<p>
                        To connect your device to the platform, use the following
                        configuration. This will need to be placed on your device.
</p>
<p class="mt-3 mb-3">
                        See the
                        <a
                            href="https://flowforge.com/docs/device-agent/register/#connect-the-device" target="_blank"
                            rel="noreferrer"
                        >Connect Your Device</a> documentation for more information.
</p>
```